src/components/DetailsPage.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+const DetailsPage = (props) => {
```

Comment:
- [ ] Each detail page should have it's unique route within the SPA so users can access it directly without getting to the home page first. I'd suggest accessing URL params with react router, here is a tiny hint on how you can do it ``<Route path="/countries/:countryName" component={detail} />``